### DISCUSSION TITLE
Is IE 11 supported in V7?

### DISCUSSION THREAD

**Discussion by mvmilanjain** [react-hook-form/react-hook-form#5266] [General]
Is IE 11 supported in V7?

Can you please share the list of supported browser in V7 and Please share demo project for IE11 with v6.

@bluebill1049

**Comment by wafflepie** [comment]
Looks like v7.13 will work fine in IE11 because `Object.defineProperty` is used instead of proxies, it's just necessary to include some polyfills and transpile RHF in your build step (e.g. by `babel-loader` in Webpack), same as with any other library which is bundled with ES6 code. I think most people who need to support IE11 don't mind these issues, though, as long as the library doesn't use anything that's not possible to polyfill, such as proxies.

@bluebill1049 Can you think of anything that could break when using RHF v7 in IE11? I've managed to implement a simple form with validations and it's working great for me so far (well, I had to implement a custom `useFormState` hook which is no longer necessary as of `v7.13.0-next.0`), but I'm not sure if there is anything more complex in RHF that could break or would require browser-specific hacking.

**Comment by jorisre** [comment]
Hi guys, I can confirm we've dropped IE11 for V7 and resolvers V2: https://github.com/react-hook-form/react-hook-form/releases/tag/v7.0.0-rc.0 _(Last line of the release note)_

Glad to see it still working.

@mvmilanjain You can find some hints to make V6 work with IE11: https://react-hook-form.com/faqs (assuming you've selected V6  on the top-left of the page)
